textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/constants/SchemaConstants.java

```diff
@@ -16,4 +16,7 @@
 
     public static final String SPAN_LIST = "spanList";
     public static final Attribute SPAN_LIST_ATTRIBUTE = new Attribute(SPAN_LIST, FieldType.LIST);
+    
+    public static final String _ID = "_id";

```

Comment:
Add a comment to the codebase to explain the purpose and assumptions about this `_id` attribute.